src/main/java/com/sarality/list/NonScrollingListViewInitializer.java

```diff
@@ -39,6 +39,11 @@ public NonScrollingListViewInitializer(Activity activity, LinearLayout listView,
     return this;
   }
 
+  public NonScrollingListViewInitializer<T, H> withEmptyList(View emptyListView) {
```

Comment:
btw, we should talk about this one in person. Having a ViewHolder that has access to the emptyView didn't make sense to me. Want to understand why we are doing this